CHANGELOG.rst

```diff
@@ -4,6 +4,15 @@ Changelog
 Current development version
 ---------------------------
 
+v2.0.0
+------
+
+- **BREAKING** Change the :code:`GraphQLDateTime` scalar type from being timezone-aware aware to
+  being timezone-naive to follow the usual database convention of naming the timezone-naive
+  type "datetime" and avoid confusion after we've added both timezone-aware and timezone-naive
+  types.
+  `#827 <https://github.com/kensho-technologies/graphql-compiler/pull/827>`__
+
```

Comment:
I think that we should just write "v.2.0.0 (upcoming release)" here.

I don't love modifying the milestone notes. One reason is that by modifying the rst text directly you can see exactly how the changelog is going to look like. Also, with the milestone notes we don't have a great way of reviewing other people's work. So if we want to distribute the changelog writing among the contributors, there is no great way to enforce the high changelog writing standards.

I think that changelog for dev release is a bit inaccurate since we've had 20 dev releases so far so it is is a bit late for that :). 

I investigated what other libraries did. The changelog entry for SQLAlchemy 2.0 already exists, but the SQLAlchemy doesn't exist yet.